tests/h/views/api/index_test.py

```diff
@@ -211,3 +206,11 @@ def test_it_returns_the_right_links_for_user_endpoints(
         assert links["user"]["update"]["url"] == (host + "/dummy/users/:username")
 
         assert set(links["user"].keys()) == set(["create", "update"])
+
+
+@pytest.fixture
```

Comment:
This is an abomination of a hack. These view tests need to be completely refactored—most of them need to be moved into the tests for `h.views.api.config` instead of being tests for the view. The config stuff here should be mocked. Anyway, will fix during implementation.

Reply:
I wonder of the existing `pyramid_config` fixture could be used?